tests/test_create.py

```diff
@@ -2488,7 +2488,7 @@ def test_conda_downgrade(
                 "--yes",
             ],
             path=prefix,
-            raise_on_error=False,
+            raise_on_error=True,
```

Comment:
I'm not sure why this change would cause the failing test to pass. Must be a bit flaky. Will try to make it less flaky.